reference-architectures/watsonx-saas-da.md

```diff
@@ -0,0 +1,148 @@
+
+---
+
+<!--
+The following line inserts all the attribute definitions. Don't delete.
+-->
+{{site.data.keyword.attribute-definition-list}}
+
+<!--
+Don't include "reference architecture" in the following title.
+Specify a title based on a use case. If the architecture has a module
+or tile in the IBM Cloud catalog, match the title to the catalog. See
+https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-naming-guidance.
+-->
+
+# Watsonx AI SaaS Automation with Assistant and Governance
+{: #title-id}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="value"}
+{: toc-use-case="value"}
+{: toc-compliance="value"}
+{: toc-version="value"}
+
+<!--
+The IDs, such as {: #title-id} are required for publishing this reference architecture in IBM Cloud Docs. Set unique IDs for each heading. Also include
+the toc attributes on the H1, repeating the values from the YAML header.
+ -->
+
+The Watsonx AI SaaS deployable architecture is designed to automate the deployment and
+configuration of the {{site.data.keyword.IBM_notm}} watsonx platform in an {{site.data.keyword.Bluemix_notm}} account. The {{site.data.keyword.IBM_notm}} watsonx platform is made of several services working together to offer AI capabilities to end users who can explore them using [{{site.data.keyword.IBM_notm}} watsonx projects](https://dataplatform.cloud.ibm.com/docs/content/wsj/manage-data/manage-projects.html?context=wx&audience=wdp). The automation also configures a starter project for an existing {{site.data.keyword.Bluemix_notm}} user.
+
+A typical use case would be setting up an {{site.data.keyword.IBM_notm}} watsonx platform ready to use in one or more Enterprise accounts granting administrator access
+to an AI Researcher for example.
+It enables an administrator to install all the services making up the {{site.data.keyword.IBM_notm}} watsonx platform, and automatically
+setup an {{site.data.keyword.IBM_notm}} watsonx project, so that the AI Researcher can log into the [platform](http://dataplatform.cloud.ibm.com/wx/home?context=wx) and start working.
+
+In more advanced use cases, the deployable architecture can be used as part of a larger solution, where it is included in a stack
+of other components. For example a solution that first setup the {{site.data.keyword.IBM_notm}} watsonx platform, and then
+installs a *chatbot application* that uses the underlying services.
+To facilitate those business challenges, the Watsonx AI SaaS deployable architecture provides output parameters that can be used programmatically
+for wiring it to the other components of the stack, and it provides the capability to install additional Watson services.
+
+## Architecture diagram
+{: #architecture-diagram}
+
+![Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture](watsonx-saas-da.svg "Architecture diagram for the Watsonx AI SaaS Automation with Assistant and Governance deployable architecture")
```

Comment:
There's 2 different names for the DA being used in the doc. Only one of them should be used:
"Watsonx AI Saas deployable architecture" or "Watsonx AI Saas Automation with Assistant and Governance deployable architecture"